RocklogicTransactions/bin/Debug/net8.0/RocklogicTransactions.runtimeconfig.json

```diff
@@ -0,0 +1,13 @@
+{

```

Comment:
Can you add a .gitignore or figure out why the current gitignore is not working . This should not be part of the commit (/bin/Debug/.. and /obj)